xds/internal/xdsclient/xdsresource/unmarshal_lds.go

```diff
@@ -152,6 +153,10 @@ func validateHTTPFilterConfig(cfg *anypb.Any, lds, optional bool) (httpfilter.Fi
 		}
 		return nil, nil, fmt.Errorf("no filter implementation found for %q", typeURL)
 	}
+	if rbac, ok := filterBuilder.(rbac.Builder); ok {
+		rbac.Name = name
+		filterBuilder = rbac
```

Comment:
This is....a nop?

Reply:
I believe if you don't do the `filterBuilder = rbac`, then the `rbac.Name` is just done to a temporary variable rather than the actual `filterBuilder`

The goal is to set this `Name` if it is an `rbac.Builder`